int16 status
int16 next
int16[] data
bool coat
bool fight
bool seat
bool towel1
bool towel2
bool warn
